tests/models/opt/test_modeling_opt.py

```diff
@@ -330,33 +330,61 @@ def test_logits(self):
         assert torch.allclose(logits, logits_meta, atol=1e-4)
 
 
-@require_tokenizers
 @slow
 class OPTGenerationTest(unittest.TestCase):
-    def setUp(self):
-        super().setUp()
-        self.all_model_path = ["facebook/opt-125m", "facebook/opt-350m"]
-
-    def test_generation(self):
-        prompts = [
+    @property
+    def prompts(self):
+        return [
             "Today is a beautiful day and I want to",
             "In the city of",
             "Paris is the capital of France and",
             "Computers and mobile phones have taken",
         ]
-        NEXT_TOKENS = [3392, 764, 5, 81]
-        GEN_OUTPUT = []
 
-        tokenizer = GPT2Tokenizer.from_pretrained("patrickvonplaten/opt_gpt2_tokenizer")
-        for model in self.all_model_path:
-            model = OPTForCausalLM.from_pretrained(self.path_model)
-            model = model.eval()
-            model.config.eos_token_id = tokenizer.eos_token_id
-
-            gen = pipeline("text-generation", model=model, tokenizer=tokenizer, return_tensors=True)
```

Comment:
better not use pipeline in modeling tests, this creates unnecessary dependencies (also for the test fetcher)